src/SOS/SOS.NETCore/SymbolReader.cs

```diff
@@ -300,8 +300,14 @@ public static void LoadNativeSymbols(SymbolFileCallback callback, IntPtr paramet
                 try
                 {
                     IEnumerable<SymbolStoreKey> keys = generator.GetKeys(flags);
-                    foreach (SymbolStoreKey key in keys)
+                    foreach (SymbolStoreKey forKey in keys)
                     {
+                        SymbolStoreKey key = forKey;
+                        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && key.FullPathName.Equals("libmscordaccore.so"))
+                        {
+                            key = new SymbolStoreKey(key.Index.Replace("libmscordaccore.so", "mscordaccore.dll"),
```

Comment:
You need to pass key.IsClrSpecialFile and key.PdbChecksums to the constructor to be complete.

Reply:
The same thing needs to happen in one of your previous changes in Analyzer.cs line 249.